src/main/java/net/fabricmc/loom/api/mappings/intermediate/IntermediateMappingsProvider.java

```diff
@@ -44,6 +44,14 @@ public abstract class IntermediateMappingsProvider implements Named {
 
 	public abstract Property<Function<String, DownloadBuilder>> getDownloader();
 
+	/**
+	 * Set to true if the minecraft version is pre 1.3, and the client and server are going to be merged.
+	 * When true the expected src namespace is intermediary, and the expected dst namespaces are clientOfficial and serverOfficial
+	 * When false the expected src namespace is named and the expected dst namespace is intermediary
+	 */
+	@ApiStatus.Experimental
+	public abstract Property<Boolean> getIsLegacyMerged();
```

Comment:
I see you reverted this as well, I think this is  the wrong name as it can still be used for the single jars.